.woodpecker.yml

```diff
@@ -111,12 +111,13 @@ pipeline:
     #   platform: linux/amd64
 
   check_diesel_schema:
-    image: jameshiew/diesel-cli
+    image: willsquire/diesel-cli
     environment:
       CARGO_HOME: .cargo
       DATABASE_URL: postgres://lemmy:password@database:5432/lemmy
     commands:
       - diesel migration run
+      - ./scripts/fix-clippy.sh
```

Comment:
Ugh, you're right, this doesn't work correctly either.

Can we just do away with this pipeline at this point? Clippy won't pass anything that diesel-cli builds because it has wildcard imports. But if you run clippy before commiting, this pipeline will fail :/ diesel-cli doesn't seem to have a way to disable wildcard imports either.

Reply:
Ah I see. In this case you need to remove `-D clippy::wildcard_imports` from this file and from `scripts/fix-clippy.sh`

Reply:
Unfortunately there seems to be no way to skip clippy on a certain file. Instead you will have to remove `-D clippy::wildcard_imports` from this file and from scripts/fix-clippy.sh